pkg/gce-cloud-provider/compute/gce.go

```diff
@@ -122,11 +122,11 @@ func generateTokenSource(configFile *ConfigFile) (oauth2.TokenSource, error) {
 
 	// DefaultTokenSource relies on GOOGLE_APPLICATION_CREDENTIALS env var being set.
 	if gac, ok := os.LookupEnv("GOOGLE_APPLICATION_CREDENTIALS"); ok {
-		klog.V(4).Infof("GOOGLE_APPLICATION_CREDENTIALS env var set %v", gac)
+		klog.V(2).Infof("GOOGLE_APPLICATION_CREDENTIALS env var set %v", gac)
```

Comment:
this doesn't actually log the token right?

Reply:
it logs the location of the token